relay.go

```diff
@@ -93,14 +93,19 @@ func (r *Relay) addLiveHop(from, to peer.ID) {
 	r.lhLk.Lock()
 	defer r.lhLk.Unlock()
 
-	r.lhCount++
 	trg, ok := r.liveHops[from]
 	if !ok {
-		trg = make(map[peer.ID]struct{})
+		trg = make(map[peer.ID]int)
 		r.liveHops[from] = trg
 	}
+	var count int
+	if count, ok = trg[to]; !ok {
+		count = 0
+	}
+	count++
```

Comment:
We can replace all of this with `trg[to]++`. That will:

1. Lookup `to` in `trg`. If it's not there, it'll return 0.
2. Increment.
3. Set `trg[to]`.

Reply:
ahhh i was actually looking into these zero values and having a hard time finding out what was up. happy to change!